backend/Exceptions/File/CsvInvalidCountFileException.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Exceptions\File;
+
+use Throwable;
+
+class CsvInvalidCountFileException extends FileException
+{
+    protected int $lineNumber;
+    protected int $expectedCount;
+    protected int $receivedCount;
+
+    public function __construct(
+        string $fullPath,
+        int $lineNumber,
+        int $expectedCount,
+        int $receivedCount,
+        int $code = 0,
+        Throwable $previous = null
+    ) {
+        $this->lineNumber = $lineNumber;
+        $this->expectedCount = $expectedCount;
+        $this->receivedCount = $receivedCount;
+        parent::__construct(
+            $fullPath,
+            $code,
+            $previous
+        );
+    }
```

Comment:
Maybe use constructor promotion?
https://stitcher.io/blog/constructor-promotion-in-php-8